src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -1144,49 +1143,6 @@ public function compileSavepointRollBack($name)
         return 'ROLLBACK TO SAVEPOINT '.$name;
     }
 
-    /**
-     * Wrap a value in keyword identifiers.
-     *
-     * @param  \Illuminate\Database\Query\Expression|string  $value
-     * @param  bool  $prefixAlias
-     * @return string
-     */
-    public function wrap($value, $prefixAlias = false)
```

Comment:
Isn't removing a public method a breaking change?

I think that removing the protected ones is too, as if someone is extending the class and calling `parent::protectedMethod()` it would break.

If this removals are needed for this feature, maybe you will need to target `master`

Reply:
I don't think this is being removed?

Reply:
I see. May bad then, sorry.

Thanks for the heads up @GrahamCampbell

Reply:
If it's not being removed, where did it go?

Reply:
I don't see much of the logic that was in this method moved anywhere?

Reply:
The parent class already had a wrap method where the only difference was the JSON support. The difference between the two has been merged into `Illuminate\Database\Grammar->wrap` therefore this override was no longer necessary. The difference which was moved can be found here https://github.com/laravel/framework/pull/36363/files#diff-13a4975d935d982e846853d4328df60c7ef014b5a4010441277b2ef4de31ed2aR67.

Reply:
Assuming the above comment answered any remaining questions I will resolve the current conversation.